### ISSUE TITLE
Add Maximum Character Limit for Phone Number Input

### ISSUE THREAD

**Issue by ahmed-gelemli** [kuasha420/react-native-paper-phone-number-input#7]
Implement a maximum character limit for phone number input. It can be:
- [E.164](https://en.wikipedia.org/wiki/E.164) defines a general format for international telephone number. Plan-conforming telephone numbers are limited to only digits and to a maximum of fifteen digits.
- It can be specific to each countries phone number length. For example: `Azerbaijan, 3 country code and 9 subscriber number: total length 12`.

**Comment by kuasha420** [comment]
Hi @ahmed-gelemli ! That's a great idea. Feel free to open a PR if you want to see it. Currently we're having a situation over here at Bangladesh. so I'm a bit limited on my time - but I'll be able to review any PRs

**Comment by ahmed-gelemli** [comment]
Thank you, I forked the repo and currently working on it. Will send PR after finished. Hoping the best for your country. 🤞

**Comment by ahmed-gelemli** [comment]
Sent PR, waiting for feedback. @kuasha420

**Comment by kuasha420** [comment]
Thank you @ahmed-gelemli for all your work in improving this little component. This is now merged and released in `v1.3.1`. 

To ensure this is not a breaking change, I made the length validation an opt in using the optional `limitMaxLength` prop. I hope you don't mind. See the readme for details. 

Thank you for your well wishes for my country. Unfortunately it's only getting worse thanks to a corrupt,  malevolent, authoritarian and oppressive government. :disappointed: 

Cheers. <3
